doc/ug/dv_methodology/index.md

```diff
@@ -185,9 +185,10 @@ The chip DV document, which is currently under active development will explain t
 ## Key Test Focus Areas
 
 When progressing through the verification stages, there are key focus areas or testing activities that are perhaps common across all DUTs.
-These are as follows.
+These are described under [Hardware Verification Stages]({{< relref "doc/project/development_stages#hardware-verification-stages-v" >}}).
+The basic staps are as follows.
 
-### Progressing towards [V1]({{< relref "doc/project/development_stages#hardware-verification-stages" >}})
+### Progressing towards V1
```

Comment:
the earlier link is wrong, perhaps we could fix the link.
```suggestion
### Progressing towards [V1]({{< relref "doc/project/development_stages#hardware-verification-stages-v" >}})
```

Reply:
I added the link in line 188. I think that is where it belongs, since it refers to a generic description of the verification stages. You can see each specific stage is an entry on a table.

A better place to link to is the V1 checklist at https://docs.opentitan.org/doc/project/checklist/#v1, and the other corresponding sections. What do you think?

Reply:
I see. Sounds good.